update chart requirements (#2876)
istio/istio.io: content/docs/setup/kubernetes/helm-install/index.md

## Installation steps

> The following commands has relative references in the Istio directory, and should be executed in the root of the Istio directory.

Reviewer: Grammar and passive voice. I suggest:
```suggestion
The following commands have relative references in the Istio directory. You must execute the commands in Istio's root directory.
```

> 

Reviewer 2: remove one space between `Istio` and `directory`

Author: done
